ddcci-multi-monitor@tim-we/files/ddcci-multi-monitor@tim-we/applet.js

```diff
@@ -82,18 +84,44 @@ class Monitor {
 
     setBrightness(value) {
         this.brightness = Math.round(value);
+        this.send_brightness = this.brightness;
         this.updateMenu();
         this.#promises = this.#promises.then(() => {
             return new Promise((resolve, reject) => {
-                Util.spawnCommandLineAsync(
-                    `ddcutil --bus=${this.bus} setvcp 10 ${this.brightness}`,
-                    resolve,
-                    reject
-                );
+                setTimeout(resolve, 5000);
+                log(`trySetBrightness start`);
```

Comment:
Do you need these `log` calls? They seem like debug values.

Reply:
Shall i remove them?